src/simulator.zig

```diff
@@ -268,9 +269,9 @@ pub const Simulator = struct {
     requests_idle: bool = false,
 
     pub fn init(allocator: std.mem.Allocator, random: std.rand.Random, options: Options) !Simulator {
-        assert(options.replica_crash_probability < 1.0);
+        assert(options.replica_crash_probability < 100.0);
         assert(options.replica_crash_probability >= 0.0);
-        assert(options.replica_restart_probability < 1.0);
+        assert(options.replica_restart_probability < 100.0);
```

Comment:
Bug, though all of the percentages were below 1 anyway.

Reply:
#2 is the plan (integers everywhere), but admittedly not a particularly high priority.
Agreed that `_probability < 100` is surprising (hence this bug), I think it ended up this way because the integer probabilities use 100 as a denominator, so the floats ended up the same.

Reply:
For context, we want to get rid of floats b/c floats aren't necessarily deterministic between different CPUs.